noxfiles/run_infrastructure.py

```diff
@@ -54,7 +55,7 @@ def run_infrastructure(
         _run_cmd_or_err(f'echo "datastores specified: {", ".join(datastores)}"')
 
     # De-duplicate datastores
-    datastores = set(datastores)
+    datastores = list(set(datastores))
```

Comment:
No problem with this, just a question. Is this because nox wants a list for the datastores?

Reply:
mypy was complaining, it was a vscode nit lol